internal/services/oracle/oracle_autonomous_database_clone_resource_test.go

```diff
@@ -0,0 +1,313 @@
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("display_name").HasValue(fmt.Sprintf("ADB%dclone", data.RandomInteger)),
+				check.That(data.ResourceName).Key("data_base_type").HasValue("Clone"),
+			),
+		},
+		data.ImportStep("admin_password", "source", "clone_type"),
+	})
+}
+
+func TestAccAutonomousDatabaseClone_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_oracle_autonomous_database_clone", "test")
+	r := AutonomousDatabaseCloneResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func TestAccAutonomousDatabaseClone_complete(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_oracle_autonomous_database_clone", "test")
+	r := AutonomousDatabaseCloneResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.complete(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("admin_password", "source", "clone_type", "refreshable_model"),
+	})
+}
+
+func TestAccAutonomousDatabaseClone_metadataClone(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_oracle_autonomous_database_clone", "test")
+	r := AutonomousDatabaseCloneResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.metadataClone(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("clone_type").HasValue("Metadata"),
+			),
+		},
+		data.ImportStep("admin_password", "source", "clone_type"),
+	})
+}
+
+func TestAccAutonomousDatabaseClone_backupTimestampLatestBackup(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_oracle_autonomous_database_clone", "test")
+	r := AutonomousDatabaseCloneResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.backupTimestampCloneLatest(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("data_base_type").HasValue("CloneFromBackupTimestamp"),
+			),
+		},
+		data.ImportStep("admin_password", "source", "clone_type", "use_latest_available_backup_time_stamp"),
+	})
+}
+
+func (r AutonomousDatabaseCloneResource) Exists(ctx context.Context, clients *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := autonomousdatabases.ParseAutonomousDatabaseID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	client := clients.Oracle.OracleClient.AutonomousDatabases
+	resp, err := client.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
+	}
+
+	return utils.Bool(resp.Model != nil), nil
```

Comment:
`utils.Bool` is deprecated

```suggestion
	return pointer.To(resp.Model != nil), nil
```